content/docs/user-guide/gems/reference/ros2/_index.md

```diff
@@ -0,0 +1,119 @@
+---
+linkTitle: ROS2
+title: ROS2 Gem
+description: The ROS 2 Gem helps to build robotic simulations with ROS 2 / Robot Operating System.
+toc: true
+---
+
+<!-- # O3DE ROS2 Gem -->
+
+This Gem enables users to develop robotic simulations through ROS2 tools and components.
+
+## Requirements
+
+* Ubuntu 20.04 or 22.04. Other Ubuntu versions and Linux distros could also work as long as they can support ROS 2.
+* [O3DE](https://www.o3de.org/)
+* Modern version of ROS 2. This instruction assumes that the `desktop` version is installed. Otherwise some packages might be missing. We support and tested with:
+  * [ROS 2 Galactic](https://docs.ros.org/en/galactic/Installation.html) with Ubuntu 20.04.
+
+#### Additional ros packages required
+
+Once the desired version of ROS 2 is installed, you will need to install additional ROS 2 packages. 
+Before running the following commands, make sure that the ROS 2 environment is setup. Replace \<distro\> with the ROS 2 distribution name (galactic, humble, ..) below and run the following command:
```

Comment:
this seems a bit out of context since we do not need to source ros to install additional packages. Perhaps the sourcing information should be added to "Requirements", for example:

Please note that to build or run projects using this Gem, you will need to have ROS sourced in your console (link to environment sourcing). The best way to ensure this is the case at all times is to add the following line to the ~/.profile file:
<sourcing line>. Note that you need to re-log for the change to take effect.